spec/05-classes-and-objects.md

```diff
@@ -133,7 +133,7 @@ consists of the following steps.
 The initialization code of an object or class (but not a trait) that follows
 the superclass
 constructor invocation and the mixin-evaluation of the template's base
-classes is passed to a special hook, which is inaccessible from user
+classes are passed to a special hook, which is inaccessible from user
```

Comment:
This is incorrect grammar. "The init code is passed."

Reply:
isn't it "init code and mixin-eval are passed?"

Reply:
I have a minute before making turkey sandwich for school, but:
```
scala> trait T { println("T") }
defined trait T

scala> class B { println("B") }
defined class B

scala> class C extends B with T with App { println("C") }
defined class C

scala> :javap -pv C
  public final void delayedEndpoint$$line5$$read$$iw$$iw$C$1();
    descriptor: ()V
    flags: (0x0011) ACC_PUBLIC, ACC_FINAL
    Code:
      stack=2, locals=1, args_size=1
         0: getstatic     #85                 // Field scala/Predef$.MODULE$:Lscala/Predef$;
         3: ldc           #86                 // String C
         5: invokevirtual #90                 // Method scala/Predef$.println:(Ljava/lang/Object;)V
         8: return
```
The constructor does `T.$init$`.